src/main/java/com/softdevsix/services/ProjectService.java

```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.services;
+
+public class ProjectService {
```

Comment:
shouldn't this be in the infrastructure layer?

Reply:
Actually, I changed it to application layer due to contains transactional logic that uses domain models.